src/Simulations/simulation.jl

```diff
@@ -100,6 +103,7 @@ function Base.show(io::IO, s::Simulation)
                      "├── Stop time: $(prettytime(s.stop_time))", "\n",
                      "├── Stop iteration: $(s.stop_iteration)", "\n",
                      "├── Wall time limit: $(s.wall_time_limit)", "\n",
+                     "├── Minimum relative step: $(s.minimum_relative_step)", "\n",
```

Comment:
```suggestion
                     "├── Minimum relative step: ", prettysummary(s.minimum_relative_step), "\n",
```

Looks nicer for round numbers